examples/VIAME_OcclusionSaliency.ipynb

```diff
@@ -0,0 +1,504 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "This notebook contains a working example to show usage of the image perturbation and classifier-based saliency map generation API with the Video and Image Analytics for Marine Environments ([VIAME](https://www.viametoolkit.org/)) Toolkit.\n",
+    "\n",
+    "This example will follow an application-like use-case where we define a functionally rigid process that transforms an input image into a number of saliency heat-maps based on our black-box classifier's output, visualizing them over the input image.\n",
+    "We will show that it is easy to change which of our API implementations are used in the application without impacting the application's successful execution, first using a sliding-window perterbation algorithm and then using the RISE perturbation algorithm.\n",
+    "\n",
+    "This will necessarilly include the use of a classification model to perform the role of the black-box classifier that we will be determining the saliency for. We will make use of the [bioharn](https://gitlab.kitware.com/viame/bioharn) package for providing classification models in PyTorch.\n",
+    "\n",
+    "### Table of Contents\n",
+    "* [The test image](#The-test-image)\n",
+    "* [The \"application\"](#The-\"application\")\n",
+    "* [Black-box Classifier](#Black-box-Classifier)\n",
+    "* [XAI Components](#XAI-Components)\n",
+    "* [Calling the Application](#Calling-the-Application)\n",
+    "\n",
+    "### Additional Python Requirements\n",
+    "The following additional python packages are required for this notebook besides the usual depedencies for `xaitk-saliency`:\n",
+    "- bioharn (v0.2.0)\n",
+    "- jupyter\n",
+    "- torch\n",
+    "- torchvision\n",
+    "\n",
+    "**Note**: To install v0.2.0 of bioharn, use the following:\n",
+    "```\n",
+    "pip install git+https://gitlab.kitware.com/viame/bioharn.git@dev/0.2.0\n",
+    "```\n",
+    "\n",
+    "### Miscellaneous\n",
+    "*Note*: When using the `from_config_dict` method below,\n",
+    "it is not unusual for the below code to throwback a warning\n",
+    "indicating missing psycopg2 and caffe modules.\n",
+    "Please disregard any such warnings.\n",
+    "\n",
+    "Reference for implementation: Zeiler MD, Fergus R. Visualizing \n",
+    "and understanding convolutional networks. InEuropean conference\n",
+    "on computer vision 2014 Sep 6 (pp. 818-833). Springer, Cham.\n",
+    "\n",
+    "License for test image used may be found in 'COCO-LICENSE.txt'."
```

Comment:
Should we update this line to reference something appropriate for the new imagery?

Reply:
Checked in with Matt- all images should be publicly available (and available on viame.kitware.com), but NOAA doesn't seem to have the concept of an image license (and it's unclear if asked, whether they will know enough to create the appropriate license). I will try to provide appropriate attribution (sans license).